docs/reference/go-style-annotation-property-practice.md

```diff
@@ -0,0 +1,16 @@
+| Synonyms                 | has_exact_synonym, has_broad_synonym, has_narrow_synonym, has_related_synonym                              | N                             | No limit     | Synonyms of the term.                                                                                              | Free text                           | database_cross_reference: reference material in which the synonymn is used                    | See [synonyms documentation](../reference/synonyms-obo.md) for guide on using synonyms                                                                     |
+| Comments                 | rdfs:comment                                                                                                     | N                             | Max 1        | Comments about the term, extended descriptions that might be useful, notes on modelling choices, other misc notes. | Free text                           | database_cross_reference: reference material relating to the comment                          | See [documentation on comments](../explanation/term-comments.md) for more information about comments                                                             |
+| Editor note              | IAO:0000116                                                                                                      | N                             | Max 1        | A note that is not relevant to front users, but might be to editors                                                | Free text                           | database_cross_reference: reference material relating to the note                             |                                                                                                                                |
+| Subset                   | [http://www.geneontology.org/formats/oboInOwl#inSubset](http://www.geneontology.org/formats/oboInOwl#inSubset)   | N                             | No limit     | A tag that marks a term as being part of a subset                                                                  | subset_property                    | None                                                                                            | See [Slim documentation](../howto/add-new-slim.md) for more information on subsets    |
+| Database Cross Reference | [http://www.geneontology.org/formats/oboInOwl#hasDbXref](http://www.geneontology.org/formats/oboInOwl#hasDbXref) | N                             | No limit     | Links out to external references.                                                                                  | Free text, see [common xref documentation](../reference/common-xref.md) | None                                                                                            | See [common xref documentation](../reference/common-xref.md) for common examples                                                                                   |
```

Comment:
Not free text!  I suggest:

The value is a string and should* take the form {prefix}:{accession}.   Often this is a compact URI (CURIE) but the prefix also often corresponds to a database and the accession to an ID in that database. See [common-xref.md](docs/reference/common-xref.md) for details.  (*Some ontologies allow full URLS in specific cases, but this is controversial).